test/correctness/mongo_model.py

```diff
@@ -451,6 +451,9 @@ def __init__(self, options, collectionname=''):
     def remove(self):
         self.data = dict()
 
+    def delete_many(self):
```

Comment:
Shouldn't `delete_many` take a filter?

Reply:
I will remove this for now. We are not using this anywhere.

Reply:
We are using it in test_core.py, line42: ` dl_collection.delete_many({})`

Reply:
we can change that line to call `remove()` instead.

Reply:
Oh never mind that's not called on an instance of MM.

Reply:
`remove()` is deprecated. PyTest doesn't like deprecated usage. Trying to avoid those warnings.